docs/basic-dsa/Strings/Advanced-techniques.md

```diff
@@ -0,0 +1,107 @@
+---
+id: Advanced String
```

Comment:
replace `id: Advanced String` with `id: advanced-string` for best practice

Reply:
i changed this